demo/src/main/java/com/novoda/demo/DataPostingModularDrm.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.demo;
+
+import android.net.Uri;
+
+import com.google.android.exoplayer2.C;
+import com.google.android.exoplayer2.upstream.DataSourceInputStream;
+import com.google.android.exoplayer2.upstream.DataSpec;
+import com.google.android.exoplayer2.upstream.DefaultHttpDataSourceFactory;
+import com.google.android.exoplayer2.upstream.HttpDataSource;
+import com.google.android.exoplayer2.util.Util;
+import com.novoda.noplayer.drm.DrmRequestException;
+import com.novoda.noplayer.drm.StreamingModularDrm;
+import com.novoda.noplayer.drm.provision.ModularDrmKeyRequest;
+
+import java.io.IOException;
+
+class DataPostingModularDrm implements StreamingModularDrm {
```

Comment:
`StreamingModularDrm` is only used by this class, can we have it implement the `DrmHandler` instead? Same with `DownloadedModularDrm`?